.pre-commit-hooks.yaml

```
@@ -0,0 +1,23 @@
+- id: selene
+  name: selene (cargo)
+  description: An opinionated Lua code linter
+  entry: selene
+  language: rust
+  types:
+      - lua
+
+- id: selene-system
+  name: selene (system)
+  description: An opinionated Lua code linter
+  entry: selene
+  language: system
+  types:
+      - lua
+
+- id: selene-docker
+  name: selene (docker)
+  description: An opinionated Lua code linter
+  entry: selene
```

```suggestion
  entry: /selene
```

This should fix the Docker run.